Command/GraphQLDumpSchemaCommand.php

```diff
@@ -12,25 +12,20 @@
 use Symfony\Component\Console\Style\SymfonyStyle;
 use Symfony\Component\DependencyInjection\ContainerInterface;
 
```

Comment:
Why this should be final ? I'm not against but think final classes just want to understand why.

Reply:
It's not a must, but I think it's important to do it when we maintain a library, this way we don't promote users to extend certain things we don't want them to. When you leave it open you create some sort of legacy and can make some users suffer to update.
Now if they want to extend it they'll have to create a new command with their behavior, which I think is for their and our advantage!